SayTheirNames/Resources/Base.lproj/Localizable.strings

```diff
@@ -3,13 +3,26 @@
   SayTheirNames
 
   Created by Kyle Lee on 6/1/20.
+  Edited by Pedro Trujillo on 6/4/20
   Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
 */
 
+/* Tab Bar */
```

Comment:
I agree that having the Strings separated by comments makes everything look more organized, but the problem is that we may be reusing these strings in other places. Cancel for example may end up on several screens but we are currently only using it in search. This then leads to the problem of moving strings around when they need to be added somewhere else.

IMO, there are only two ways to approach the organization of these strings that will allow the app to scale:
1. We continue to separate strings into sections, but use dot syntax so the generated strings can be grouped together when accessing them.
Example:
```
/* Search */
"search.cancel" = "Cancel";

/* Home */
"home.cancel" = "Fancy Cancel";
```
```
label.text = Strings.Search.cancel
label.text = Strings.Home.cancel
```

2. We keep all the strings alphabetized by their key value and simply add more descriptive keys for different use case.
Example:
```
"cancel" = "Cancel";
"cancel_fancy" = "Fancy Cancel" // we could also do "fancy_cancel"
```
```
label.text = Strings.cancel
label.text = Strings.cancelFancy
```